src/openpersonen/api/data_classes/persoon.py

```diff
@@ -10,3 +12,32 @@ class Persoon:
     geheimhoudingPersoonsgegevens: bool
     naam: Naam
     geboorte: Geboorte
+
+    backend_function_name = None
+
+    @classmethod
+    def list(cls, bsn):
+        class_instances = []
+        function = getattr(backend, cls.backend_function_name)
```

Comment:
rename to `func` to prevent conflicts.

```suggestion
        func = getattr(backend, cls.backend_function_name)
```